### DISCUSSION TITLE
Vim friendly layouts

### DISCUSSION THREAD

**Discussion by KYTAZ0** [manna-harbour/miryoku#80] [Q&A] [answered]
Hey, greetings thanks for sharing this. It really inspired me and got me into a 3w6 split keyboard which just arrived and hopefully I'll be rocking it soon.

I saw that there are a lot of different user configurations in the repo here 
https://github.com/manna-harbour/miryoku_qmk/tree/miryoku/users
so I thought I'd ask if you're aware of modifications to the original layout which are intended to be friendlier to vim usage.

Also something I'm not entirely sure about; in those keyboard types (36key split) where do your fingers rest by convention? is it M or N the key your index points at per se?

Thanks, your project is truly magnificent

**Comment by KYTAZ0** [comment]
Spending a bit of time and digging into the documentation probably is sufficient to answer the question so its not necessary to take the time I think, unless you have something explicit to input of course :)

**Comment by manna-harbour** [comment, marked as answer]
Great, good luck with the new board! Here's a build workflow run for 3w6 with a few options: https://github.com/manna-harbour/miryoku_qmk/actions/runs/2148347156

For vi, the main issue is that the home row mods mean that you can't hold hjkl to move. Instead you can use the cursor keys on Nav, and there's an option to have them in the hjkl positions. That's also useful if you use non-QWERTY alphas with vi. There's also escape on the Base layer on a tertiary thumb. For other vi commands on non-QWERTY try to use the mnemonics rather than position.

With 36 keys (3x5+3) the middle row is the home row and the index home is on Colemak T and N, and the home thumb is the middle thumb key. On the diagram, those 4 keys are marked with a homing bar, although it's not very obvious. The index finger moves across to reach the inner index columns, but those keys are not used as frequently. All other digits only move 1 position in each direction along a single axis.

Thanks!
